### PR TITLE
add documentation for validate-downloads job

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: jenkins/doc/jobs/validate-downloads.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  latest release for all of the active LTS releases and the Current release
+  will be validated. If you select `latest` then the
+  latest release for all of the active LTS releases, the Current release
+  as well as the nightlies will be validated.
+  If you select 'vX' where X is is for a stream (ex v8, v7, v4) then the
+  latest release from that stream will be validated.
+  If you select a fully qualified version (ex v8.4.0) then that specific
+  release will be validated.
+* `DownloadLocation`: This controls whether the release or nightly download
+  location is used. For example f you want to validate the latest v8 nightly
```
Comment (richardlau) [inline jenkins/doc/jobs/validate-downloads.md]: `if`

Comment (mhdawson) [reply jenkins/doc/jobs/validate-downloads.md]: Fixed

**Comment by mhdawson** [comment]
@nodejs/build any comments and if not can I get an approval so I can land.

File: jenkins/doc/jobs/validate-downloads.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# validate-downloads
+
+This job validates that the downloads on nodejs.org are good. 
+It is scheduled to be run nightly and can also be run manually
+after a release is made. **Note:** there is delay between
+when releases are generated and when they will be available on
+nodejs.org(up to 15 minutes) so releasers may have to wait
```
Comment (refack) [inline jenkins/doc/jobs/validate-downloads.md]: ARM builds can take up to 60 minutes.

File: jenkins/doc/jobs/validate-downloads.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  If you select 'vX' where X is is for a stream (ex v8, v7, v4) then the
+  latest release from that stream will be validated.
+  If you select a fully qualified version (ex v8.4.0) then that specific
+  release will be validated.
+* `DownloadLocation`: This controls whether the release or nightly download
+  location is used. For example if you want to validate the latest v8 nightly
+  you would set StreamAndVersion to 'v8' and DownloadLocation to 'nightly'.
+
+If validation fails an email notification is sent to the
+`download-validation-alert` email alias. If you would like to get these
```
Comment (refack) [inline jenkins/doc/jobs/validate-downloads.md]: updated to `release-validation-alert`

**Comment by refack** [comment]
@mhdawson I added my nit fixes as a `[suggestion]` commit. Obviously feel free to push it out.

**Comment by gibfahn** [comment]
I'll review properly later today, but as an initial thought I'd rather we didn't start having separate documentation of these jobs, as in my experience it just gets out of date.

I'd rather we document the script clearly with comments, and then have the parameters just be the arguments to the script. If the way the parameter names map to the script is clear (it's either an environment variable, or it's passed in directly as an argument to the script) then that should be self-documenting. We could then have a single file that has an overview of each of the jobs to document what we have (i.e. the first paragraph of this doc).

I'll try to go through this in the next couple of hours and come up with something concrete.

**Comment by mhdawson** [comment]
@gibfahn waiting on your follow up input.

**Comment by gibfahn** [comment]
So I'd suggest having a single file `jenkins/jobs.md` with an entry for each job, and a link to the job. Then we document the job parameters in the job itself (otherwise we have to duplicate that info, people probably won't think to look here).

#### `jenkins/jobs.md:`

<br/>

    # Jenkins Jobs
    
    ## [validate-downloads](https://ci.nodejs.org/view/All/job/validate-downloads/)
    
    This job validates that the downloads on nodejs.org are good. It is scheduled to
    be run nightly and can also be run manually after a release is made. **Note:**
    there is delay between when releases are generated and when they will be
    available on nodejs.org (up to 60 minutes) so releasers may have to wait a bit
    be able to complete the validation.
    
    If validation fails an email notification is sent to the
    `release-validation-alert` email alias. If you would like to get these
    notifications submit a PR to have your email added for that alias in
    https://github.com/nodejs/email/blob/master/iojs.org/aliases.json.
    
    This job needs to be updated each time a we roll over to a new Current
    release.

**Comment by refack** [comment]
@gibfahn can land?

**Comment by gibfahn** [comment]
Okay, pushed a commit to my branch that does what I suggested in earlier comments.

LGTM.

**Comment by mhdawson** [comment]
Landed as aed300d66bab013e50829ef1e62ddce792461597

**Pull request closed without merging**